lib/deb_utils.rb

```diff
@@ -6,9 +6,12 @@ def self.extract_deb(file, target)
     #   target: (optional) a specified file to extract from archive, should be passed in regex or string
     #
     # Example:
-    #   extract_deb('example.deb', 'test.txt') # extract `test.txt` from example.deb
-    #   extract_deb('example.deb', /data\..*/) # extract files from example.deb with filenames matching the /data\..*/ regex
-    #   extract_deb('example.deb', /*/)        # extract all files from example.deb
+    #   extract `test.txt` from example.deb
+    #   extract_deb('example.deb', 'test.txt')
```

Comment:
We need a way to tell rubocop to not touch any of the code comments😅

Reply:
I think adding this to `.rubocop.yml` would handle that?

```
AllowedPatterns: ['^ *# '] 
```

Reply:
That was my editing, to deal with line length. It's not a great solution, but the best one I could think of.

Reply:
I honestly think we should leave line length edits. just tell rubocop to not worry about it.

Reply:
Line length is a big part of code style. How about ignoring line length on comments?